packages/venia-ui/lib/components/Gallery/item.js

```diff
@@ -126,6 +126,7 @@ class GalleryItem extends Component {
                 alt={name}
                 width={imageWidth}
                 height={imageHeight}
+                loading="lazy"
```

Comment:
This is so nice but also [not very widely supported](https://caniuse.com/#feat=loading-lazy-attr). 

I'd also much rather see it implemented in our `Image` component than in one single use of `img`.

Reply:
> This is so nice but also not very widely supported.

It's also backwards compatible so it won't break anything! 

I actually just did this to play with the performance optimization aspect of including **a lot** of new images through Page Builder. We can look to improve this aspect of the Gallery when we visit our Products story.